i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/releases/release-notes.md

```diff
@@ -8,6 +8,50 @@ import TabItem from '@theme/TabItem';
 
 ## 2025
 
+### April
+
+<Tabs groupId="sync">
+<TabItem value="New connectors" label="New connectors">
+
+- [**Exense Step REST API**](../procedures/applications-exense-step-restapi.md) - Initial release of Exense Step REST API.
+
+</TabItem>
+<TabItem value="Enhancements" label="Enhancements">
+
+- [**APC UPS SNMP**](../procedures/hardware-ups-apc-snmp.md) - Enhance replace last time output to be humanly readable in **battery-status mode**, from a [The Watch idea](https://thewatch.centreon.com/ideas/hardware-ups-apc-snmp-plugin-warn-crit-output-with-options-warning-replace-lasttime-critical-replace-lasttime-3427).
+- [**Colubris SNMP**](../procedures/network-colubris-snmp.md) - Added the option `--check-overload` in **uptime** service template.
+- [**F5 BigIP SNMP**](../procedures/network-loadbalancers-f5-bigip-snmp.md) - Added new **cpu-usage** mode, based on community contribution [PR 4701](https://github.com/centreon/centreon-plugins/pull/4701).
+- [**Fortinet FortiMail SNMP**](../procedures/network-fortinet-fortimail-snmp.md) - Added the option `--check-overload` in **uptime** service template.
+- [**Fortinet FortiSwitch SNMP**](../procedures/network-fortinet-fortiswitch-snmp.md) - Added the option `--check-overload` in **uptime** service template.
+- [**Fortinet Fortigate Rest API**](../procedures/network-fortinet-fortigate-restapi.md) - Added a new mode **certificates**, from community contribution [PR 5473](https://github.com/centreon/centreon-plugins/pull/5473).
+- [**Fortinet Fortigate SNMP**](../procedures/network-firewalls-fortinet-fortigate-snmp.md) - Added **switch-usage** mode, **switch discovery rule** and **vdom discovery rule**, from community contribution [PR 5474](https://github.com/centreon/centreon-plugins/pull/5474).
+- [**HP-UX SNMP**](../procedures/operatingsystems-hpux-snmp.md) - Added the option `--check-overload` in **uptime** service template.
+- [**IP-Label Ekara Rest API**](../procedures/applications-monitoring-iplabel-ekara-restapi.md) - Steps are now displayed in the actual steps order instead of alphabetical.
```

Comment:
```suggestion
- [**IP-Label Ekara Rest API**](../procedures/applications-monitoring-iplabel-ekara-restapi.md) - Steps are now displayed in the actual step order instead of alphabetical.
```